drivers/veteran_confirmation/app/models/veteran_confirmation/va_check_history.rb

```diff
@@ -0,0 +1,91 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module VeteranConfirmation
+  class VaCheckHistory < GrdaWarehouseBase
+    include NotifierConfig
+
+    self.table_name = :va_check_histories
+
+    # Response values
+    CONFIRMED = 'confirmed'.freeze
+    NOT_CONFIRMED = 'not confirmed'.freeze
+
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
+
+    scope :most_recent_first, -> { order(check_date: :desc) }
+
+    validates :response, presence: true
+
+    def occured_within(days)
+      check_date.present? && check_date <= Date.current - days
+    end
+
+    def check
+      query_result = self.class.check(::GrdaWarehouse::Hud::Client.where(id: client_id))[client_id]
+      return unless query_result.present?
+
+      update!(
+        response: query_result,
+        check_date: Date.current,
+      )
+
+      result = query_result == CONFIRMED
+      client.update!(va_verified_veteran: client.va_verified_veteran? || result)
```

Comment:
if `client.va_verified_veteran?` is true, we shouldn't be here right?
maybe up on the first line:
`return if client.va_verified_veteran?`